src/components/shared/formItems/ProjectReserved.tsx

```diff
@@ -64,14 +66,16 @@ export default function ProjectReserved({
     <Form.Item
       extra={
         <>
-          <TabDescription>
-            <Trans>
-              Initial issuance rate will be {formattedNum(initialIssuanceRate)}{' '}
-              tokens / ETH for contributors.{' '}
-              {formattedNum(initialReservedTokensPerEth)} tokens / ETH will be
-              reserved by the project.
-            </Trans>
-          </TabDescription>
+          {isCreate && (
```

Comment:
6.) Hide all the messages about 'initial' token issuance on the reconfig modal since they aren't relavent anymore: